### PR TITLE
Laravel 6 support

### PR DESCRIPTION
**Pull request by petrenkorf**
Fixed composer.json to accept Laravel 6 packages as dependencies.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "name": "bkwld/laravel-haml",
+    "name": "petrenkorf/laravel-haml",
```
Comment (weotch) [inline composer.json]:
```suggestion
    "name": "bkwld/laravel-haml",
```

File: src/Bkwld/LaravelHaml/ServiceProvider.php [hunk 2]
```diff
@@ -69,6 +69,7 @@ public function boot() {
 		switch($this->version()) {
 			case 4: $this->bootLaravel4(); break;
 			case 5: $this->bootLaravel5(); break;
+            case 6: $this->bootLaravel5(); break;
```
Comment (weotch) [inline src/Bkwld/LaravelHaml/ServiceProvider.php]:
```suggestion
			case 6: $this->bootLaravel5(); break;
```

**Review by weotch — CHANGES_REQUESTED** [review]
Small tweaks

**Comment by dogaoz** [comment]
It would be great if this could be merged so that we can use this on latest laravel 6 LTS.

File: src/Bkwld/LaravelHaml/ServiceProvider.php [hunk 3]
```diff
@@ -144,7 +145,13 @@ public function registerHamlBladeCompiler() {
 	 * @return array 
 	 */
 	public function getConfig() {
-		$key = $this->version() == 5 ? 'laravel-haml' : 'laravel-haml::config';
+        $configFiles = [
+            4 => 'laravel-haml::config',
+            5 => 'laravel-haml',
+            6 => 'haml'
```
Comment (weotch) [inline src/Bkwld/LaravelHaml/ServiceProvider.php:151]: 👍

**Pull request merged**